### PR TITLE
Add manufactured solution to problem

### PR DESCRIPTION
**Pull request by kronbichler**
I now compute some right hand side (sine with different frequencies in different directions and vector components) which should be a good test for correctness. @peterrum if you agree you can continue to put the test into the other directories. One thing I have not yet figured out is why we build two executables, one called `test` in the basic directory and then another executable in `test.release/test.release` to be used for the test.

### CODE REVIEW

**Review by peterrum — APPROVED** [review]

**Comment by peterrum** [comment]
> One thing I have not yet figured out is why we build two executables, one called test in the basic directory and then another executable in test.release/test.release to be used for the test.

The motivation to have a second executable in each folder is the following: here we can test different degrees, smaller problem sizes, multiple s in the case of s-step CG, and don't get any timings. I guess we could get the same result in an other way but this was the most straight forward to accomplish since I only had to rely on deal.II build infrastructure.

File: common_code/poisson_operator.h [hunk 1]
```diff
@@ -114,9 +114,8 @@ namespace Poisson
           all_indices_uniform.resize(Utilities::pow(3, dim) * data->n_cell_batches(), 1);
         }
 
-      FE_Nothing<dim> dummy_fe;
-      FEValues<dim>   fe_values(*data_->get_mapping_info().mapping,
-                              dummy_fe,
+      FEValues<dim> fe_values(*data_->get_mapping_info().mapping,
+                              data_->get_dof_handler().get_fe(),
```
Comment (peterrum) [inline common_code/poisson_operator.h:118]: This should not have had happened! Let me check what went wrong on dealii/master.

Comment (kronbichler) [reply common_code/poisson_operator.h:118]: I think the error is fine. It was introduced by the geometry commit in #10 I believe.

Comment (peterrum) [reply common_code/poisson_operator.h:118]: Oh, right! I interpreted the error differently. I thought the new constructor of `FE_Nothing` makes problems...

**Pull request merged**